openlibrary/utils/solr.py

```diff
@@ -77,6 +77,16 @@ def get_many(
         ).json()
         return [doc_wrapper(doc) for doc in resp['response']['docs']]
 
+    def update_in_place(
+        self,
+        request,
+    ):
+        resp = requests.post(
+            f'{self.base_url}/update?update.partial.requireInPlace=true,commit=true',
```

Comment:
```suggestion
            f'{self.base_url}/update?update.partial.requireInPlace=true&commit=true',
```

I think this might need to be an `&` here; good luck 😁

Reply:
I have no idea how this ever worked, and I'm at least 90% sure that at one point, it did. Resolved!